### DISCUSSION TITLE
I cannot start or push stuff to repo any idea?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45202] [Repositories]
### Select Topic Area

Product Feedback

### Body

So, I have been trying to do so like in the title says, and so far no luck, it says a message like fatal: detected dubious ownership in repository at 'D:/SourceControl/Github/Optimized Pawn/OptimusPawn'
'D:/SourceControl/Github/Optimized Pawn/OptimusPawn' is on a file system that doesnot record ownership
To add an exception for this directory, call:
And so it is impossible to do so, I think that is not cool, any idea?

**Comment by jge162** [comment]
@Neonghostone The error message is telling you that the file system on your local machine does not record ownership, and that you need to add an exception for the directory you are trying to clone the repository into.

One possible solution to this issue would be to try cloning the repository into a different directory on your local machine, one that is on a file system that does record ownership.

**Reply by ghost** [reply]
Hello one thing to solve it was getting it to 'D:/SourceControl/Github/Optimized Pawn/OptimusPawn' is on a file system that doesnot record ownership
To add an exception for this directory, I did run this command  git config --global --add safe.directory D:/SourceControl/Github/LSF2NormalGitOldDepra/PotatoPrroject Official something like that..
